arviz/tests/base_tests/test_data_zarr.py

```diff
@@ -0,0 +1,103 @@
+import importlib
+import os
+import shutil
+from collections.abc import MutableMapping
+
+import numpy as np
+import pytest
+import zarr
```

Comment:
This won't work because it will raise an exception here before getting to the skipif